sensor_msgs/Image image
---
ObjectArray objects

